ui/src/app/views/dataset-page.tsx

```diff
@@ -687,7 +696,7 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
                               justifyContent: 'center',
                               lineHeight: '32px',
                             }}>
-                       {previewRow.domain}
+                       {previewRow.domain === 'OBSERVATION' ? 'SURVEY' : previewRow.domain}
```

Comment:
Same thing, should we just move all 'OBSERVATION' to be 'SURVEY'? Are they identical?

Reply:
Ok so Concept set with domain Survey creation was failing as it was not able to find Survey corresponding OMOP table. Will check with Jon if we can use maybe observation table corresponding to Domain Survey? If yes will update accordingly